### DISCUSSION TITLE
Cross compilation for installers

### DISCUSSION THREAD

**Discussion by Selemba1000** [jreleaser/jreleaser#1693] [Q&A]
Does jreleaser support the creation of native installers or executables on a different system than the target? I'm searching for a solution to create for example windows installers/executables from my Java/Kotlin code from my Linux dev environment or my Linux ci/cd. It would be really nice if I wouldn't have to source Linux, windows and Mac machines to build my cross platform java app. Package managers are not an option, because of limitations of the users.

**Comment by aalmiray** [comment]
No, it does not. Building executables is a job for a build tool. JReleaser is a release tool, although it does provide assemblers for GraalVM native images. 

However, you may build platform specific executables on GitHub Actions. Here are some examples:

https://jreleaser.org/guide/latest/examples/java/index.html

https://jreleaser.org/guide/latest/examples/miscellaneous/index.html

**Reply by Selemba1000** [reply]
I see. So we still don't have a reasonable way of packaging and compiling from just one os. Let's hope that changes soon.

**Reply by aalmiray** [reply]
Well it depends, not for Java that is. Go and Rust have that problem solved.
